taskvine/src/bindings/python3/ndcctools/taskvine/manager.py

```diff
@@ -245,13 +245,13 @@ def staging_directory(self):
                 path.mkdir(parents=True, exist_ok=True)
             return self._staging_explicit
         else:
-            return cvine.vine_get_runtime_path_staging(self._taskvine, None)
+            return cvine.vine_get_path_staging(self._taskvine, None)
 
     ##
     # Get the caching directory of the manager
     @property
     def cache_directory(self):
-        return cvine.vine_get_runtime_path_caching(self._taskvine, None)
```

Comment:
Is there a reason we use the staging directory instead of the cache directory? I probably missed out on why this change was here..